source/WebApp/tests/component.codemirror.mode-asm-instructions.tests.ts

```diff
@@ -0,0 +1,42 @@
+import CodeMirror from 'codemirror';
+import '../components/internal/codemirror/mode-asm';
+
+test('mode-asm-instructions codemirror highlights correctly', async () => {

```

Comment:
Since you are testing full `mode-asm` (good choice btw!), I think the file name and test name does not have to contain `-instructions`.

```suggestion
test('mode-asm codemirror highlights correctly', async () => {
```
^ also the file name

Reply:
I also reworded it little explicitly what this test is trying to achieve.